### PR TITLE
fix(dart/transform): Consider of line numbers in inliner_for_test

### LINKED ISSUE #5281: Angular transformer is adding extra lines to Dart files -- the line number in stack traces do not match our source code
The Angular transformer is Component templates in the Dart files as a string of HTML.

This is causing the line numbers reported in stack traces to be incorrect.

Steps to reproduce:

pub serve test <test target>

In Dartium, open the component's Dart file, notice that the file numbers are shifted.

============

Suggested fix: include the HTML and CSS as strings at the end of the file.  Reference those strings from the @Component, taking care not to alter the line count.

### PR DESCRIPTION
**Pull request by kegluneq**
Ensure that line numbers aren't changed by inliner_for_test.

Fixes #5281

### CODE REVIEW

File: modules_dart/transform/lib/src/transform/inliner_for_test/transformer.dart [hunk 1]
```diff
@@ -154,14 +179,17 @@ class _ViewPropInliner extends RecursiveAstVisitor<Object> {
     _writer.print('styles: const [');
     for (var url in urls) {
       if (url is String) {
-        _writer.print("r'''");
-        _writer.asyncPrint(_readOrEmptyString(url));
-        _writer.print("''', ");
+        final inlineKey = '${_inlinedValueBase}${_inlinedValues.length}';
```
Comment (jbdeboer) [inline modules_dart/transform/lib/src/transform/inliner_for_test/transformer.dart]: add "Style" to the key for readability: _vStyle0

**Comment by jbdeboer** [comment]
Thanks for the quick fix. The team and our stack traces appreciate it !

**Comment by kegluneq** [comment]
Updated the inline value names to `_template$num` and `_style$num`, plus updated to provide print order guarantees since the unit tests depend on it.

PTAL

**Comment by kegluneq** [comment]
Ready for commit - thanks!
/cc @vsavkin

**Pull request closed without merging**